spanner/spanner_snippets/spanner/spanner_create_backup.go

```diff
@@ -47,7 +47,13 @@ func createBackup(w io.Writer, db, backupID string) error {
 
 	// Get the name, create time and backup size.
 	createTime := time.Unix(backup.CreateTime.Seconds, int64(backup.CreateTime.Nanos))
-	fmt.Fprintf(w, "Backup %s of size %d bytes was created at %s\n", backup.Name, backup.SizeBytes, createTime.Format(time.RFC3339))
+	versionTime := time.Unix(backup.VersionTime.Seconds, int64(backup.VersionTime.Nanos))
```

Comment:
The Go client uses the auto-generated admin client for almost all operations, __except__ `CreateBackup`. There is a hand-written function for `CreateBackup` that currently does not support setting a version time. We should probably look into this more in general and not just add a `VersionTime` parameter to the function, as it would be a breaking change, and it would mean that we would add additional parameters in the future if additional properties are added to the `Backup` proto.